### PR TITLE
Fix:  Minor design and sdk bugs

### LINKED ISSUE #2588: 💄 Various Design Bugs
### Subject of the issue

- [ ] extra space appearing between label and number on mobile

bug

- [ ] another issue i'm seeing - table headers should have a lighter background everywhere(only on perps rn) and the first cell(odd numbers) should always be darker. the last table row should have a border bottom. 

- [ ] another issue - the border on drop down cell items should be removed

https://github.com/Kwenta/kwenta/assets/548702/a5d1b36c-93cf-41fe-ace0-c8f0d36c4d78

- [ ] also noticed this "synthetic" being prefixed to synth assets kinda weird. maybe we should get rid of it altogether

- [ ]  stats, docs, support icons still missing 

- [ ] we should get rid of this tiny share icon on mobile

### PR DESCRIPTION
**Pull request by LeifuChen**
## Description
- [X] Fixed the `suggestDecimals` bug in `formatPercent` https://github.com/Kwenta/kwenta/blame/dev/packages/sdk/src/utils/number.ts#L184
- [X] Updated the market details on mobile
- [X] Fixed the extra space appearing between the label and number on mobile
截屏2023-07-19 01 27 03

- [X] Remove the extra border on the `funding period toggle` dropdown
截屏2023-07-19 01 27 33

- [X] Remove the prefix `synthetic` of the token selector in the exchange page
截屏2023-07-19 01 28 58

- [X] Remove the tiny share icon on mobile
截屏2023-07-19 01 28 19

- [ ] Add stats (@asherism Missing icon for stats), docs, and support icons into the footer
截屏2023-07-19 01 29 43

- [X] On the markets page, the table headers should have a lighter background, and the first row should always be darker. The last table row should display the bottom border line.
截屏2023-07-19 01 30 08

## Related issue
Closes #2588

### CODE REVIEW

File: packages/sdk/src/utils/number.ts [hunk 1]
```diff
@@ -181,7 +181,7 @@ export const formatPercent = (
 	value: WeiSource,
 	options?: { minDecimals?: number; suggestDecimals?: boolean; maxDecimals?: number }
 ) => {
-	let decimals = suggestedDecimals ? suggestedDecimals(value) : options?.minDecimals ?? 2
+	let decimals = options?.suggestDecimals ? suggestedDecimals(value) : options?.minDecimals ?? 2
```
Comment (avclarke) [inline packages/sdk/src/utils/number.ts:184]: Good catch

File: packages/app/src/sections/futures/MarketDetails/MarketDetails.tsx [hunk 2]
```diff
@@ -167,6 +174,7 @@ const MarketSkew: React.FC<MarketDetailsProps> = memo(({ mobile }) => {
 					<MarketDetailValue
 						color="green"
 						value={formatPercent(marketInfo ? marketInfo?.openInterest.longPct : 0, {
+							suggestDecimals: false,
```
Comment (avclarke) [inline packages/app/src/sections/futures/MarketDetails/MarketDetails.tsx:177]: I don't think this is necessary

File: packages/app/src/sections/futures/MarketDetails/MarketDetails.tsx [hunk 3]
```diff
@@ -175,6 +183,7 @@ const MarketSkew: React.FC<MarketDetailsProps> = memo(({ mobile }) => {
 					<MarketDetailValue
 						color="red"
 						value={formatPercent(marketInfo ? marketInfo?.openInterest.shortPct : 0, {
+							suggestDecimals: false,
```
Comment (avclarke) [inline packages/app/src/sections/futures/MarketDetails/MarketDetails.tsx:186]: Same here

File: packages/app/src/sections/futures/MarketDetails/MarketDetails.tsx [hunk 4]
```diff
@@ -118,7 +118,10 @@ const DailyChangeDetail: React.FC<MarketDetailsProps> = memo(({ mobile }) => {
 			dataKey={MarketDataKey.dailyChange}
 			value={
 				indexPriceWei.gt(0) && pastPrice?.rate
-					? formatPercent(indexPriceWei.sub(pastPrice.rate).div(indexPriceWei) ?? ZERO_WEI)
+					? formatPercent(indexPriceWei.sub(pastPrice.rate).div(indexPriceWei) ?? ZERO_WEI, {
+							suggestDecimals: false,
```
Comment (avclarke) [inline packages/app/src/sections/futures/MarketDetails/MarketDetails.tsx:122]: It's false by default

**Review by avclarke — APPROVED** [review]
Good work catching the formatPercent bug

File: packages/app/src/sections/futures/MarketDetails/MarketDetails.tsx (hunk 4 above)
Comment (LeifuChen) [reply packages/app/src/sections/futures/MarketDetails/MarketDetails.tsx]: Yes. That's true. Removed.

File: packages/app/src/sections/futures/MarketDetails/MarketDetails.tsx (hunk 3 above)
Comment (LeifuChen) [reply packages/app/src/sections/futures/MarketDetails/MarketDetails.tsx]: Removed. Thanks.

File: packages/app/src/sections/futures/MarketDetails/MarketDetails.tsx (hunk 2 above)
Comment (LeifuChen) [reply packages/app/src/sections/futures/MarketDetails/MarketDetails.tsx]: Yes. Removed. Thanks.

**Pull request merged**